docs/guide/http_cache.md

```diff
@@ -607,6 +607,17 @@ Note that the system does not add this tag to responses itself, just purges if p
 Response tagging using this tag is currently meant to be done inline in the template logic / views
 based on your decision.
 
+!!! tip Troubleshooting with contents tagged by the huge number of tags (too long headers)
+    
+    In case of complex contents, like for instance Landing Pages with many blocks, you might get into trouble with too long PURGE request `xkey` header what might cause not all of necessary cache entries being cleared. You can see `502 Headers too long` errors then.
```

Comment:
It's currently not a problem for PURGE actually, or should not be, as we do one purge per tag atm: https://github.com/ezsystems/ezplatform-http-cache/blob/master/src/PurgeClient/VarnishPurgeClient.php#L37-L49

However that will change once we move to FOSHttpCache v2.

Anyway, it's for now more a issue for responses I think, afaik tags being stripped off before cache is stored and tagged by xkey, meaning they won't be invalidated by the tags that got stripped.

Reply:
Yes, you are totally right, changed in https://github.com/ezsystems/developer-documentation/pull/454/commits/e4af94272995a573996d03e4b8469c20862b8a40.